core/rff_weather_indeces.mcf

```diff
@@ -0,0 +1,15 @@
+Node: heavyPrecipitationIndex
+dcid: dcs:heavyPrecipitationIndex
```

Comment:
This can be just:

```
Node: dcid:heavyPrecipitationIndex
```
(without "dcid" line)

which is equivalent to:

```
Node: heavyPrecipitationindex
dcid: "heavyPrecipitationIndex"
```